pedalboard/__init__.pyi

```diff
@@ -0,0 +1,297 @@
+# This stub file was automatically generated by mypy's `stubgen`,
+# The file was then modified manually; if you get a build failure
+# that leads you here, feel free to modify the file manually as well.
+#
+# This file may diverge from the runtime/native implementation of the code,
+# as long as the interface remains the same. (i.e.: pure-Python classes
+# may be listed here even if they override/inherit from native classes.)
+
+from typing import Any, ClassVar, Dict, Iterator, List, Optional, Type, Union, overload
+
+import numpy
+from numpy.typing import NDArray
```

Comment:
Just a note, this may not be compatible with older versions of numpy/python. Numpy is currently unpinned so it's probably fine, but existing with TF<=2.7 may be an issue for example.

I think to include this you'll have to pin numpy in the repo to >=1.21 (maybe 1.20 I can't remember)